src/controllers/EventController.ts

```diff
@@ -1,8 +1,17 @@
-import { JsonController, Param, Get, Post, Delete, Body, OnUndefined } from 'routing-controllers';
+import {
+  JsonController,
+  Param,
+  Get,
+  Post,
+  Delete,
+  Body,
+  OnUndefined,
+  UseBefore,
+} from 'routing-controllers';
 import { singleton, inject } from 'tsyringe';
 import { ResponseSchema } from 'routing-controllers-openapi';
 
-import { Event } from '@Entities';
+import { Event, AppUserRole } from '@Entities';
```

Comment:
is this an enum?

Reply:
didnt see it defined anywhere

Reply:
Yeah it's an enum in AppUserEntity

Reply:
cool - was a little confused as I thought it was an entity